x/ecocredit/server/core/features/msg_update_class_fee.feature

```diff
@@ -0,0 +1,134 @@
+Feature: Msg/MsgUpdateClassFees
+
+  The class fees can be updated:
+  - when the authority is a governance account address
+  - when the provided class fees are valid
```

Comment:
This is handled already with basic message validation. The server tests should focus on state validation and expected behavior so no need to duplicate the scenario here.